Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalEEAlgo.cc

```diff
@@ -22,7 +22,7 @@
 #include "Geometry/HGCalCommonData/interface/HGCalTypes.h"
 #include "Geometry/HGCalCommonData/interface/HGCalWaferType.h"
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```

Comment:
Shouldn't debug messages be turned off in the master?

Reply:
Done